make/config/pebble/chart/templates/configmap.yaml

```diff
@@ -22,7 +22,17 @@ data:
         "externalAccountMACKeys": {
           "kid-1": "zWNDZM6eQGHWpSRTPal5eIUYFTu7EajVIoguysqZ9wG44nMEtx3MUAsUDkMTQ12W"
         },
-        "domainBlocklist": ["google.com"]
+        "domainBlocklist": ["google.com"],
+        "profiles": {
+          "123h": {
+            "description": "Certificates with duration 123h",
+            "validityPeriod": {{ mul 60 60 123 }}
+          },
+          "456h": {
+            "description": "Certificates with duration 456h",
+            "validityPeriod": {{ mul 60 60 456 }}
+          }
+        }
```

Comment:
I meant to add a note about these, I'll do it in a followup:

>   NOTE: Two profiles are configured but only one is used in the E2E tests. A
>   second profile is added to demonstrate that Pebble selects a random
>   profile unless the client specifies one. Tests which do not specify a
>   profile must expect to receive 123h or 456h certificates.

https://github.com/letsencrypt/pebble/pull/491#issuecomment-2926689390